pkg/gitreceive/build.go

```diff
@@ -167,6 +167,7 @@ func build(
 			conf.PodNamespace,
 			appConf.Values,
 			slugBuilderInfo.TarKey(),
+			gitSha.Short(),
```

Comment:
You're right that it isn't required for docker builds, but I thought that providing parity is useful as it means the `SOURCE_VERSION` variable exists in the build env whenever code is built as a result of a `git push deis`. I will certainly be using this extra functionality but I am happy to break it out into a separate commit or separate PR for further discussion, let me know.